src/FirstSteps/RPG/HeroesRepository.cs

```diff
@@ -0,0 +1,26 @@
+﻿using FirstSteps.RPG.Heroes;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+
+namespace FirstSteps.RPG
+{
+    public class HeroesRepository
+    {
+        const string FilePath = @"C:\Users\misza\source\repos\first-steps\src\FirstSteps\RPG\File\hero.json";
```

Comment:
You should use [relative paths](https://learn.microsoft.com/en-us/dotnet/standard/io/file-path-formats) because when you run the app on another machine probably `C:\Users\misza\source\repos` will not be present.